node/lib/cmd/checkout.js

```diff
@@ -45,9 +45,17 @@ exports.helpText = `Check out a commit in the meta-repository and each
  */
 exports.configureParser = function (parser) {
 
+    parser.addArgument(["-b"], {
+        dest: "new branch name",
+        help: `Create a new branch to check out.`,
+        nargs: 1,
+    });
+
     parser.addArgument(["committish"], {
         type: "string",
         help: "commit to check out",
+        defaultValue: null,
+        require: false,
```

Comment:
I think its "required", and also I believe it is invalid (doesn't work) for positional args.